addon/index.js

```diff
@@ -39,13 +37,13 @@ Mustache.parse(templates.experimentList);
 const Metrics = require('./lib/metrics');
 const survey = require('./lib/survey');
 const WebExtensionChannels = require('./lib/webextension-channels');
+const ToolbarButton = require('./lib/toolbar-button');
 
-const PANEL_WIDTH = 300;
-const FOOTER_HEIGHT = 50;
-const EXPERIMENT_HEIGHT = 80;
 const INSTALLED_PANEL_WIDTH = 250;
 const INSTALLED_PANEL_HEIGHT = 56;
 
+const EXPERIMENT_UPDATE_INTERVAL = 24 * 60 * 60; // 1 day
```

Comment:
This comes out to `86,400 milliseconds` which converts to `86.4 seconds`.
To check once a day, we want `86,400,000 milliseconds`

So if you just update this to
``` js
const EXPERIMENT_UPDATE_INTERVAL = 24 * 60 * 60 * 1000; // 1 day
```
that should do the trick.